### PR TITLE
EC2 Information for system tests added 

### PR DESCRIPTION
**Pull request by pro-akim**
|Related issue|
|-------------|
|   #4527   |

## Description

Adding information to `wazuh-qa/tests/system/README.md` about EC2 requirements for running system tests in EC2

### CODE REVIEW

File: tests/system/README.md [hunk 1]
```diff
@@ -4,7 +4,25 @@ Wazuh - System quality assurance automation templates
...
+| Environment                  | EC2                                       |
+|------------------------------|-------------------------------------------|
+|Basic_cluster                 |Ubuntu 22.04.2 LTS `C5`.`XLarge` `15GB` HD |
+|Big_cluster_40_agents         |Ubuntu 22.04.2 LTS T3.Large `60GB` HD      |
+|Agentless_cluster             |Ubuntu 22.04.2 LTS T3.Large 30GB HD        |
+|Four_manager_disconnected_node|Ubuntu 22.04.2 LTS T3.Large 30GB HD        |
+|One_manager_agent             |Ubuntu 22.04.2 LTS T3.Large 30GB HD        |
+|Manager_agent                 |Ubuntu 22.04.2 LTS T3.Large 30GB HD        |
+|Enrollment_cluster            |Ubuntu 22.04.2 LTS T3.Large 30GB HD        | 
+| Basic_environment            |Ubuntu 22.04.2 LTS T3.Large 30GB HD        |
```
Comment (rauldpm) [inline tests/system/README.md:22]: AWS does not provide HD but HDD, also remove the extra space in line 33 between | and Basic

Why the 60GB is encapsulated between `` and 30GB no? The same goes for the type of instance.

File: tests/system/README.md [hunk 2]
```diff
@@ -4,7 +4,25 @@ Wazuh - System quality assurance automation templates
 
 ## Setting up a test environment
 
-To run these tests we need to use a **Linux** machine and install the following tools:
+To run these tests we need to use a **Linux** machine.
```
Comment (rauldpm) [inline tests/system/README.md:7]: Avoid the use of `we`, try to use impersonal pronouns like:

> To run these tests it is needed to use a **Linux** machine.

**Review by rauldpm — CHANGES_REQUESTED** [review]
Minor changes

File: tests/system/README.md (hunk 1 above)
Comment (pro-akim) [reply tests/system/README.md]: Done in https://github.com/wazuh/wazuh-qa/pull/4536/commits/ebbcdb5657a2c91573e29a50775b12441c2473bd

File: tests/system/README.md [hunk 3]
```diff
@@ -4,7 +4,25 @@ Wazuh - System quality assurance automation templates
 
 ## Setting up a test environment
 
-To run these tests we need to use a **Linux** machine and install the following tools:
+To run these tests it will be required to use a **Linux** machine.
```
Comment (verdx) [inline tests/system/README.md:7]: "A linux machine will be required"

File: tests/system/README.md [hunk 4]
```diff
@@ -4,7 +4,25 @@ Wazuh - System quality assurance automation templates
...
+|Big_cluster_40_agents         |Ubuntu 22.04.2 LTS T3.Large 60GB SSD       |
+|Agentless_cluster             |Ubuntu 22.04.2 LTS T3.Large 30GB SSD       |
+|Four_manager_disconnected_node|Ubuntu 22.04.2 LTS T3.Large 30GB SSD       |
+|One_manager_agent             |Ubuntu 22.04.2 LTS T3.Large 30GB SSD       |
+|Manager_agent                 |Ubuntu 22.04.2 LTS T3.Large 30GB SSD       |
+|Enrollment_cluster            |Ubuntu 22.04.2 LTS T3.Large 30GB SSD       | 
+|Basic_environment             |Ubuntu 22.04.2 LTS T3.Large 30GB SSD       |
+
+
+Now it will be needed to install the following tools:
```
Comment (verdx) [inline tests/system/README.md:25]: "The following tools will need to be installed"

**Review by verdx — CHANGES_REQUESTED** [review]

File: tests/system/README.md (hunk 4 above)
Comment (pro-akim) [reply tests/system/README.md]: Done in https://github.com/wazuh/wazuh-qa/pull/4536/commits/72c6457ae1bdb34e0a5c08492263a7ea8caca7f2

File: tests/system/README.md (hunk 3 above)
Comment (pro-akim) [reply tests/system/README.md]: Done in https://github.com/wazuh/wazuh-qa/pull/4536/commits/b30b1e13cf55de1b3faa64fea6350554f9914d8b

**Review by verdx — APPROVED** [review]
LGTM

**Review by juliamagan — APPROVED** [review]
LGTM

**Pull request merged**